qiskit/scheduler/utils.py

```diff
@@ -34,3 +39,81 @@ def format_meas_map(meas_map: List[List[int]]) -> Dict[int, List[int]]:
         for q in sublist:
             qubit_mapping[q] = sublist
     return qubit_mapping
+
+
+def measure(qubits: List[int],
+            backend: Optional['BaseBackend'] = None,
+            inst_map: Optional[InstructionScheduleMap] = None,
+            meas_map: Optional[Union[List[List[int]], Dict[int, List[int]]]] = None,
+            qubit_mem_slots: Optional[Dict[int, int]] = None) -> Schedule:
+    """
+    Return a schedule which measures the requested qubits according to the given
+    instruction mapping and measure map, or by using the defaults provided by the backend.
+
+    By default, the measurement results for each qubit are trivially mapped to the qubit
+    index. This behavior is overridden by qubit_mem_slots. For instance, to measure
+    qubit 0 into MemorySlot(1), qubit_mem_slots can be provided as {0: 1}.
+
+    Args:
+        qubits: List of qubits to be measured.
+        backend: A backend instance, which contains hardware-specific data required for scheduling.
+        inst_map: Mapping of circuit operations to pulse schedules. If None, defaults to the
+                  ``circuit_instruction_map`` of ``backend``.
+        meas_map: List of sets of qubits that must be measured together. If None, defaults to
+                  the ``meas_map`` of ``backend``.
+        qubit_mem_slots: Mapping of measured qubit index to classical bit index.
+
+    Returns:
+        A schedule corresponding to the inputs provided.
+
+    Raises:
+        PulseError: If both ``inst_map`` or ``meas_map``, and ``backend`` is None.
+    """
+    schedule = Schedule(name="Default measurement schedule for qubits {}".format(qubits))
+    try:
+        inst_map = inst_map or backend.defaults().instruction_schedule_map
+        meas_map = meas_map or backend.configuration().meas_map
+    except AttributeError:
+        raise PulseError('inst_map or meas_map, and backend cannot be None simultaneously')
+    if isinstance(meas_map, List):
+        meas_map = format_meas_map(meas_map)
+
+    measure_groups = set()
+    for qubit in qubits:
+        measure_groups.add(tuple(meas_map[qubit]))
+    for measure_group_qubits in measure_groups:
+        if qubit_mem_slots is not None:
+            unused_mem_slots = set(measure_group_qubits) - set(qubit_mem_slots.values())
+        default_sched = inst_map.get('measure', measure_group_qubits)
```

Comment:
just to be sure, `measure` is general name for measurement sequence? if some provider send a instruction map with e.g. `meas` for measurement, this program will crash.

Reply:
yes, you're right but I'm unaware of anything we can do to mitigate this :( we could try a few like 'meas' and 'MEAS' and 'MEASURE' and 'M' etc if this fails, but it's dependent on the backend. @taalexander ?

Reply:
I guess we could at least enclose in a try catch block and raise a more helpful error message if `'measure'` is not found

Reply:
or add **kwarg with `measure` as default?

Reply:
good idea, I think both of the above. Since we are given the backend, we can use it to include the basis gates in the error message and explain how to fix the error if it occurs

Reply:
I'll push the fix soon. 😊